packages/components/pie-toggle-switch/src/defs.ts

```diff
@@ -1,6 +1,14 @@
 import { RTLComponentProps } from '@justeattakeaway/pie-webc-core';
 
-export interface ToggleSwitchProps extends RTLComponentProps {
+export type LabelProps = {
+    text: string;
+    position: {
+        leading: boolean;
+        trailing: boolean;
```

Comment:
what do you think about having it as position=[leading, trailing]?
i believe this will bring more clarity than having to flip the boolean logic for each

Reply:
I was thinking this too, ill take a look

Reply:
Thanks, this actually helped clean up a bit of code too 👍